igvc_ws/src/igvc_serial/package.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>igvc_serial</name>
+  <version>0.0.0</version>
+  <description>The igvc_serial package</description>
+
+  <!-- One maintainer tag required, multiple allowed, one person per tag -->
+  <!-- Example:  -->
+  <!-- <maintainer email="jane.doe@example.com">Jane Doe</maintainer> -->
+  <maintainer email="kichang@todo.todo">kichang</maintainer>
```

Comment:
Not an issue, but would be good to figure out how we want to do maintainers and licenses

Reply:
We should probably just have a maintainer for whoever worked on the package so it is easy to know who to ask questions about the package too. The license should just be MIT project-wide. I updated the project.xml to match this